lib/src/app/app.dart

```diff
@@ -92,15 +271,178 @@ class AppClient {
   }
 
   /// Get a stream of [LogEntry] for a specific [RobotPart]. Logs are sorted by descending time (newest first)
-  Stream<List<LogEntry>> tailLogs(RobotPart part, {bool errorsOnly = false}) {
+  Stream<List<LogEntry>> tailLogs(RobotPart part, {bool errorsOnly = false, String? filter}) {
```

Comment:
`part` should probably just be `String partId`

Reply:
Similar to my previous response lol